ext/curl/curl.stub.php

```diff
@@ -50,19 +50,19 @@ function curl_multi_exec(CurlMultiHandle $multi_handle, &$still_running): int {}
 
 function curl_multi_getcontent(CurlHandle $multi_handle): ?string {}
 
-/** @param int|null $msgs_in_queue */
-function curl_multi_info_read(CurlMultiHandle $multi_handle, &$msgs_in_queue = null): array|false {}
+/** @param int $queued_message_count */
+function curl_multi_info_read(CurlMultiHandle $multi_handle, &$queued_message_count = null): array|false {}
```

Comment:
Descriptive, but a bit long. Would simply `$message_count` be sufficient?  Or `int $queued_messages`